src/test/java/uk/gov/justice/laa/crime/cfecrime/utils/FullMeansTestOutcomeTest.java

```diff
@@ -0,0 +1,105 @@
+package uk.gov.justice.laa.crime.cfecrime.utils;
+
+import org.junit.Ignore;
+import org.junit.jupiter.api.Test;
+import uk.gov.justice.laa.crime.cfecrime.cma.enums.CaseType;
+import uk.gov.justice.laa.crime.cfecrime.cma.enums.FullAssessmentResult;
+import uk.gov.justice.laa.crime.cfecrime.cma.enums.MagCourtOutcome;
+import uk.gov.justice.laa.crime.cfecrime.cma.enums.MeansTestOutcome;
+import uk.gov.justice.laa.crime.cfecrime.cma.response.AssessmentResult;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+@Ignore
+public class FullMeansTestOutcomeTest {
+
+    private MeansTestOutcome fullMeansTestOutcome;
+
+    private CaseType caseType;
+    private FullAssessmentResult result;
+
+    @Test
+    public void givenAssessmentResult_WhenCaseType_Null_ThenIsNotPossible() {
+        //for code coverage only
+        //instantiates class
```

Comment:
This feels like an anti-pattern which pollutes this first test. As this class has no state, should we not create a private constructor or something so that this problem goes away?